13/umbraco-deploy/getting-started/deploy-settings.md

```diff
@@ -214,6 +216,14 @@ If you have deleted languages that have already existing translations, you may w
 
 When this is in place a translation for a language that doesn't exist in the target environment will be ignored. A warning message will be output to the log.
 
+### SetEmptyDictionaryItemsOnTransfer
+
+When deploying dictionary items, Umbraco Deploy will follow the approach used for all content, which is that empty values are transferred and set.
+
+As such, if you transfer a dictionary item with an empty translation to another environment that already contains a translation, it will be overwritten.
```

Comment:
```suggestion
When deploying dictionary items, Umbraco Deploy follows the approach used for all content, emptying values that are transferred and set.

If you transfer a dictionary item with an empty translation to another environment that already contains a translation, it will be overwritten.
```